projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -63,7 +63,11 @@ import { MultiSelectJustify } from './multi-select-justify';
                 <span *ngIf="triggerValues.selectedItemsCount > 1" class="trigger-more-items"
                   >+{{ triggerValues.selectedItemsCount - 1 }}</span
                 >
-                <ht-icon class="trigger-icon" icon="${IconType.ChevronDown}" size="${IconSize.Small}"></ht-icon>
+                <ht-icon
+                  class="trigger-icon"
+                  icon="${IconType.ChevronDown}"
+                  [size]="this.size === '${IconSize.ExtraSmall}' ? '${IconSize.ExtraSmall}' : '${IconSize.Small}'"
```

Comment:
why not use `this.size` directly?

Reply:
because I would be modifying the regular case that already uses Small size as a regular

Reply:
`[size]="this.size"` should also work right?

Reply:
hmm.. you are comparing and assigning`SelectSize` with IconSize. that seems incorrect.

Reply:
This component also has iconSize btw.

Reply:
thanks, done